CHANGELOG.md

```diff
@@ -97,7 +102,16 @@ of response and not the raw API endpoint response
   - **CUMULUS-2634**
     - Changed `sfEventSqsToDbRecords` Lambda to use new upsert helpers for executions, granules, and PDRs
     to ensure out-of-order writes are handled correctly when writing to Elasticsearch
-    
+  - **CUMULUS-2510**
+    - Removed `stream_enabled` and `stream_view_type` from `collections_table`
+      TF definition.
+    - Removed `aws_lambda_event_source_mapping` TF definition on collections
+      DynamoDB table.
```

Comment:
We usually create another `### Removed` header to list the deletions. You also deleted the `publishCollections` Lambda itself

Reply:
[`754f871` (#2432)](https://github.com/nasa/cumulus/pull/2432/commits/754f871e6b2a2231b24c1dc75a24f09bf623fabd)